index.md

```diff
@@ -1,10 +1,15 @@
 ---
 layout: home
 conference_ad_row:
```

Comment:
@dnilasor do you mean to keep the previous summit-row name? No big deal if you don't change it.

Reply:
I did because it seemed like a generic-enough row name. Maybe I should change it to an event more generic name (promo_row ?) so we don't need to change it? @dorbin maybe there is some value there